VKappProject/VKappProject/Constants.swift

```diff
@@ -89,4 +90,41 @@ enum Constants {
         static let rotationAngle: CGFloat = -90
```

Comment:
```suggestion
        static let rotationAngle = -90
```
Явная аннотация типо